WowPacketParser/Parsing/Parsers/QuestHandler.cs

```diff
@@ -1019,11 +1069,64 @@ public static void HandleQuestRequestItems434(Packet packet)
             }
 
             // flags, if any of these flags is 0 quest is not completable
-            packet.ReadUInt32("Unk flags 1"); // 2
-            packet.ReadUInt32("Unk flags 2"); // 4
-            packet.ReadUInt32("Unk flags 3"); // 8
-            packet.ReadUInt32("Unk flags 4"); // 16
-            packet.ReadUInt32("Unk flags 5"); // 64
+            QuestStatusFlags statusFlag1 = packet.ReadUInt32E<QuestStatusFlags>("StatusFlags1"); // 2
+            QuestStatusFlags statusFlag2 = packet.ReadUInt32E<QuestStatusFlags>("StatusFlags2"); // 4
+            QuestStatusFlags statusFlag3 = packet.ReadUInt32E<QuestStatusFlags>("StatusFlags3"); // 8
+            QuestStatusFlags statusFlag4 = packet.ReadUInt32E<QuestStatusFlags>("StatusFlags4"); // 16
+            QuestStatusFlags statusFlag5 = packet.ReadUInt32E<QuestStatusFlags>("StatusFlags5"); // 64
+
+            QuestStatusFlags statusFlags = statusFlag1 | statusFlag2 | statusFlag3 | statusFlag4 | statusFlag5;
```

Comment:
i don't think this is correct, ORing 5x 32 bit flags into 32bit seems super weird. These are 160 possible flags afterall.

Reply:
Client logically ANDs these in 15595 and never uses the split values. No idea about live